tests/Http/ActionStatsTest.php

```diff
@@ -141,4 +144,37 @@ public function testLocationFilter()
         $response->assertStatus(200);
         $this->assertEquals(1, $decodedResponse['meta']['pagination']['count']);
     }
+
+    /**
+     * Test expected results for filtering by group ID and after cursor.
+     *
+     * @return void
+     */
+    public function testGroupTypeIdFilterAndCursor()
+    {
+        // Create five action stats.
+        $actionStats = factory(ActionStat::class, 5)->create();
+        // Create a group with our first action stat's school.
+        $group = factory(Group::class)->create([
+            'school_id' => $actionStats[0]->school_id,
+        ]);
+
+        // Find a cursor to search by.
+        $response = $this->getJson($this->url);
+
+        $decodedResponse = $response->decodeResponseJson();
+        $firstCursor = $decodedResponse['data'][0]['cursor'];
+
+        // Verify no errors are thrown when additionally filtering by group_type_id.
```

Comment:
This is exactly what I was thinking of for a test! 💯

Reply:
Re: your question in the PR, I think if you had swapped around the order of the commits this test would have failed before the changes introduced in 264c776 right? If so, this test sounds like exactly what we need here to make sure we don't "undo" the fix you've applied here in the future!